rad/src/error.rs

```diff
@@ -707,7 +710,17 @@ mod tests {
         // So just try all the possible `u8` values and return the successful ones
         (0u8..=255).filter_map(|error_code| {
             match RadonErrors::try_from_primitive(error_code) {
-                Ok(x) => Some(x),
+                Ok(x) => {
+                    if x == RadonErrors::BridgeMalformedRequest
+                        || x == RadonErrors::BridgeTooManyWitnesses
+                        || x == RadonErrors::BridgePoorIncentives
+                    {
+                        // We skip these RadonErrors because there are not related with the protocol
+                        None
+                    } else {
+                        Some(x)
+                    }
+                }
```

Comment:
For the fans of `match` xD
```suggestion
                Ok(RadonErrors::BridgeMalformedRequest )
                | Ok(RadonErrors::BridgeTooManyWitnesses)
                | Ok(RadonErrors::BridgePoorIncentives) {
                    // We skip these RadonErrors because there are not related with the protocol
                    None
                },
                Ok(x) => Some(x),
```